lib/ansible/modules/windows/win_mapped_drive.py

```diff
@@ -20,12 +20,19 @@
 description:
 - Allows you to modify mapped network drives for individual users.
 notes:
-- This can only map a network drive for the current executing user and does not
-  allow you to set a default drive for all users of a system. Use other
-  Microsoft tools like GPOs to achieve this goal.
 - You cannot use this module to access a mapped drive in another Ansible task,
   drives mapped with this module are only accessible when logging in
   interactively with the user through the console or RDP.
+- To create a mapped drive with saved credentials, use become to access the
+  current user's credential store when creating the drive. Any credentials
+  should be added to the store with C(cmdkey.exe).
+- Using the I(username) and I(password) options will allow a module to create
+  a mapped drive that requires authentication but the drive will not accessible
+  on the next interactive logon as the password is not saved.
+- This module will only map a drive for the current user, to map a drive for
```

Comment:
I think this would read better as 2 sentences, viz:
By default, this module will only map a drive for the current user.  To map a drive for all users of a system, use beomce with the U(SYSTEM) account.